.github/workflows/test.yml

```diff
@@ -417,12 +428,14 @@ jobs:
       with:
         path: ~/Library/Caches/lima/download
         # hashFiles do not seem to support symlinks
-        key: ${{ runner.os }}-${{ hashFiles('examples/experimental/vz.yaml') }}
+        key: ${{ runner.os }}-${{ hashFiles('examples/*.yaml') }}
     - name: Make
       run: make
     - name: Install
       run: make install
     - name: Install test dependencies
       run: brew install qemu bash coreutils
     - name: Test
-      run: ./hack/test-templates.sh templates/experimental/vz.yaml
+      env:
+        ARGS: "--vm-type vz --mount-type virtiofs --rosetta --network vzNAT"
```

Comment:
```suggestion
        LIMACTL_CREATE_ARGS: "--vm-type vz --mount-type virtiofs --rosetta --network vzNAT"
```

Reply:
(These args will no longer be needed after merging:
- https://github.com/lima-vm/lima/pull/1951
)